ui/rogue/combat/sim.ts

```diff
@@ -187,12 +188,41 @@ const SPEC_CONFIG = registerSpecConfig(Spec.SpecCombatRogue, {
 	],
 });
 
+function getMasterySoftCapConfig(): StatCap {
```

Comment:
Please make this an arrow function per modern ES6 standards;
`const getMasterySoftCapConfig = () => : StatCap`